tools/gulpfile.js

```diff
@@ -72,7 +72,11 @@ gulp.task('coverage', function () {
         ' cover' +
         ' --include-all-sources' +
         ' --dir coverage' +
-        ' -x "bin/** -x doc/** -x specs/** -x coverage/** -x index.js -x gulpfile.js"' +
+        ' -x "doc/**"' +
+        ' -x "specs/**"' +
+        ' -x "coverage/**"' +
+        ' -x "gulpfile.js"' +
+        ' -x "index.js"' +
```

Comment:
We aren't skipping the `bin` directory.  And bin files can be programmatically unit tested just like the rest of the library so there's no reason to exclude them.